scripts/updatekinto.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+const got = require("got");
+
+const SERVER = "https://settings-writer.prod.mozaws.net/v1";
+const CID = "fxmonitor-breaches";
+const DEBUG = true;
+
+async function run() {
+  const RemoteSettingsBreachesSet = new Set((await got(
+    "https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/fxmonitor-breaches/records",
+    { json: true }
+  )).body.data.map(b => b.Name));
+
+  const hibp_breaches = (await got(
+    "https://haveibeenpwned.com/api/v2/breaches",
+    { json: true }
+  )).body;
+
+  const new_breaches = [];
+
+  for (const breach of hibp_breaches) {
+    if (breach.IsSpamList || breach.IsRetired || !breach.IsVerified || !breach.Domain) {
+      continue;
+    }
+
+    if (RemoteSettingsBreachesSet.has(breach.Name)) {
+      continue;
+    }
+
+    new_breaches.push(breach);
+  }
+  for (const breach of new_breaches) {
+    const data = {
+      Name: breach.Name,
+      Domain: breach.Domain,
+      BreachDate: breach.BreachDate,
+      PwnCount: breach.PwnCount,
+      AddedDate: breach.AddedDate,
+    };
+
+    if (DEBUG) {
+      console.log(data);
+      continue;
+    }
+
+    try {
+      await got.post(`${SERVER}/buckets/main-workspace/collections/${CID}/records`, {
+        headers: {
+          "Content-Type": "application/json",
+          "authorization": "Bearer <insert token here>",
```

Comment:
I saw your commit message,

> _"This needs to be updated to use credentials from the environment instead of a pasted bearer token."_

But not sure if we're planning on doing that in this PR or a future PR and we should slap a `// TODO` comment on this. Although I'm not sure if we're really tracking TODO comments in this repo or if we should remove that ESLint rule, since it may just be adding noise to the CI logs.